docs/user/trusted-publishers/creating-a-project-through-oidc.md

```diff
@@ -65,7 +65,7 @@ provide the name of the PyPI project that will be created.
     Setting up ActiveState to create a PyPI project is the same as
     updating a project. You need to provide the name of the ActiveState project,
     the ActiveState organization that project belongs to, and the ActiveState user who will be performing
-    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/PYPI).
+    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/pypi).
```

Comment:
This is 404 as well, is that expected?

Reply:
Looks like this should be:
```suggestion
    the publish action. Learn more about getting set up on the ActiveState Platform [here](https://docs.activestate.com/platform/start/pypi/).
```

Reply:
It's working for me...I see you suggested a change below to add the trailing slash though...and now that I look, my browser ADDS that 🤦 .  Fixing.